src/main/java/jp/co/soramitsu/sora/crypto/common/SignatureTypeEnum.java

```diff
@@ -0,0 +1,25 @@
+package jp.co.soramitsu.sora.crypto.common;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import jp.co.soramitsu.crypto.ed25519.EdDSAEngine;
+import jp.co.soramitsu.crypto.ed25519.EdDSASecurityProvider;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum SignatureTypeEnum {
```

Comment:
add annotation @FieldDefaults, in this case you could to remove private and final keywords